common/reviews/api/debug-certificate-manager.api.md

```diff
@@ -16,9 +16,11 @@ export class CertificateManager {
 // @public
 export class CertificateStore {
     constructor();
-    certificateData: string | undefined;
-    readonly certificatePath: string;
-    keyData: string | undefined;
+    get certificateData(): string | undefined;
+    set certificateData(certificate: string | undefined);
```

Comment:
@sachinjoseph FYI upgrading from TypeScript 3.5 -> 3.7 introduces these getter/setters that make the library unusuable by every project that uses a compiler older than 3.5.  Is that what you intended?

See [this thread](https://twitter.com/octogonz_/status/1217565789252800512) for details.

Reply:
We don't want that I guess. I will revert the RSC changes.